src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,442 @@ authors: [oluoluoxenfree, alextait1]
+
+Accessibility is not the sole responsibility of developers to implement. The entire product team and organization have to have it as part of their accountabilities in order to succeed. Accessibility work needs to shift left in the product cycle, meaning it needs to be baked into the research, ideation and design stages before it is developed. 
+
+### Potential accessibility responsibilities by role 
+
+This list is not exhaustive and is intended to encourage thought about how all of the roles can work together to achieve accessible websites and applications, like a relay race of accountability.
+
+- Human Resources/People Ops: 
+  - Recruiting and hiring people with accessibility skills including disabled practitioners.
+  - Creating an inclusive work environment where people’s disabilities are accommodated.
+- UX /Product designers: 
+  - Considering and talking to people with a  range of disabilities in the research and ideation stages.
+  - Annotating wireframes with accessibility information such as intended heading hierarchy, skip links, alternative text suggestions (which could also come from copywriters/content folks) and screen reader only text.
+- UI designers: 
+  - Color contrast choices, font selections, spacing and line height considerations.
+  - Animation considerations (determining if they are necessary, supplying static assets for prefers-reduced-motion scenarios, designing pause/stop mechanisms).
+- Product managers:
+  - Prioritizing accessibility work in the roadmap, ensuring it does not become technical debt at the end of a backlog.
+  - Creating processes for teams to validate their work such as including accessibility in the definition of done and acceptance criteria.
+- Developers:
+  - Preferring native HTML solutions whenever possible, understanding ARIA and when to use it.
+  - Validating all work with automated and manual testing, evaluating colleagues’ pull requests with the same criteria.
+- Quality Assurance
+  - Including accessibility testing in their workflow.
+  - Advocating for accessibility considerations when contributing to the team’s quality strategy and acceptance criteria.
+- Leadership/C-Suite
+  - Giving employees bandwidth to learn and grow their accessibility skillset and hiring practitioners with expertise and lived experiences.
+  - Considering accessibility core to the product outcomes and viewing accessibility excellence as promotable work.
+
+The tech industry needs to move towards inclusion driven development. Although this requires some up-front investment, it is much easier and likely less expensive over time to build accessibility into the entire cycle such that it can be baked into the product rather than trying to retrofit sites and apps that were constructed without it in mind.
```

Comment:
```suggestion
The tech industry needs to move towards inclusion-driven development. Although this requires some up-front investment, it is much easier and likely less expensive over time to build accessibility into the entire cycle such that it can be baked into the product rather than trying to retrofit sites and apps that were constructed without it in mind.
```